salt/modules/git.py

```diff
@@ -1782,14 +1782,18 @@ def ls_remote(cwd=None,
         .. versionchanged:: 2015.8.0
             Argument renamed from ``repository`` to ``remote``
 
-    ref : master
-        The name of the ref to query. Can be a branch or tag name, or the full
-        name of the reference (for example, to get the hash for a Github pull
-        request number 1234, ``ref`` can be set to ``refs/pull/1234/head``
+    ref
+        The name of the ref to query. Optional, if not specified, all refs are
+        returned. Can be a branch or tag name, or the full name of the
+        reference (for example, to get the hash for a Github pull request number
+        1234, ``ref`` can be set to ``refs/pull/1234/head``
 
         .. versionchanged:: 2015.8.0
             Argument renamed from ``branch`` to ``ref``
 
+        .. versionchanged:: 2015.8.?
```

Comment:
Update this to ``2015.8.4``. This is the next release slated in the 2015.8 release cycle.